network-api/networkapi/mozfest/templates/partials/primary_heroguts.html

```diff
@@ -2,18 +2,26 @@
 
 <div id="hero">
   <div class="banner">
-    {% if page.banner %}
-      {% if homepage %}
-        {% include "../fragments/hero/large.html" with page=page %}
-        {% include "./homepage_banner.html" with page=page %}
-      {% else %}
-        {% include "../fragments/hero/normal.html" with page=page %}
-      {% endif %}
+    {% if homepage %}
+      {% include "../fragments/hero/large.html" with page=page %}
+      {% include "./homepage_banner.html" with page=page %}
     {% else %}
-    <img src="/_images/mozfest/Festival Banner 1.jpg" alt="">
+      {% include "../fragments/hero/normal.html" with page=page %}
     {% endif %}
   </div>
 
+  {% if homepage %}
+  <div class="container">
+    <div class="row full-width-cutout">
+      <div class="content col-12 pt-4">
+        <h1 class="h2-heading mb-0">{% if page.header %}{{ page.header }}{% else %}{{ page.title }}{% endif %}</h1>
```

Comment:
https://redpen.io/vuefbd37973f0c2a38
From the mockup it seems like this heading on "Mozfest homepage" will be long (e.g., `MozFest 2019 from 21 to 27 October. London, England, UK.`) and will change over time. I think it would make more sense if by default we display `header` of the page instead of `title`.